packages/vcs/.eslintrc.cjs

```diff
@@ -0,0 +1,50 @@
+/** @type {import('eslint').ESLint.ConfigData} */
+module.exports = {
```

Comment:
Eventually we can make use a global linter for the entire codebase?

Reply:
ah you mean like a single `.eslintrc.cjs` at the root for the codebase?

Reply:
done in this PR: https://github.com/TBD54566975/web5-js/pull/120 thanks to @frankhinek